src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -37,4 +42,158 @@ def read_fhir_bundles(container_url: str, container_prefix: str) -> Iterator[dic
 
 
 def upload_bundle_to_fhir_server(bundle: dict) -> None:
-    pass
+    """Upload a FHIR bundle to FHIR server"""
+    process_fhir_resource(bundle)
+
+
+def process_fhir_resource(
+    fhir_json: dict = {}, fhir_string: str = "", fhir_filepath: str = ""
+):
+    """Process a file containing FHIR resource(s) enclosed by ndjson
+    :param str filename: FHIR resource(s) enclosed by ndjson
+    """
+    if len(fhir_json) > 0:
+        pass
+    elif os.path.exists(fhir_filepath):
+        fhir_string = open(fhir_filepath).read()
+        fhir_json = json.loads(fhir_string)
+    else:
+        fhir_json = json.loads(fhir_string)
+
+    import_to_fhir(fhir_json)
+
+
+def import_to_fhir(fhir_json: dict, method: str = "PUT"):
+    """Import a FHIR resource to the FHIR server.
+    The submissions may Bundles or individual FHIR resources.
+
+    See :py:func:`_ensure_bundle_batch` for details about Bundle
+    conversion and handling.
+
+    :param dict fhir_json: FHIR resource in json format.
+    :param str method: HTTP method to use (currently PUT or POST supported)
+    """
+    try:
+        token = get_access_token()
```

Comment:
This won't work with the CDC azure (they got rid of this access method) - check `get_blob_client` with the `DefaultAzureCredential` bit

Reply:
Auth updated and tested locally using the Azure CLI in conjunction with updated python code.